src/App.tsx

```diff
@@ -1,16 +1,23 @@
 import './App.scss';
+import React, { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
+
+export const App: React.FC = () => {
+  const [movies, setMovies] = useState<Movie[]>(moviesFromServer);
+  const addMovie = (newMovie: Movie) => {
+    setMovies([...movies, newMovie]);
```

Comment:
use callBack with previous state
```suggestion
    setMovies(prevMovies => [...prevMovies, newMovie]);
```

Reply:
isn't fixed